src/utils/api.ts

```diff
@@ -6,6 +6,12 @@ import { matchPath } from 'react-router'
 import { paths } from './apiSchema'
 import authStore from './authStore'
 
+for (const key of ['REACT_APP_API_URL', 'REACT_APP_API_KEY']) {
```

Comment:
I moved the two checks into a for loop. In the future, I suppose this could be moved up to validate the presence and completeness of the .env file more generally.